index.css

```diff
@@ -1,3 +1,18 @@
-html {
-  
+html {}
+.board { margin:1em auto; border-collapse:collapse }
+.board td {
+    cursor:pointer;
+    width:100px; height:100px;
```

Comment:
I would put these on separate lines... it's too easy to miss `height` if you're just skimming through the list.